src/test/regress/expected/isolation_metadata_sync_vs_all.out

```diff
@@ -608,6 +728,415 @@ t
 (1 row)
 
 
+starting permutation: s2-create-type s1-begin s2-begin s1-start-metadata-sync s2-drop-type s1-commit s2-commit s3-compare-snapshot
+step s2-create-type:
+ CREATE TYPE my_type AS (a int, b int);
+
+step s1-begin:
+    BEGIN;
+
+step s2-begin:
+ BEGIN;
+
+step s1-start-metadata-sync:
+ SELECT start_metadata_sync_to_node('localhost', 57638);
+
+start_metadata_sync_to_node
+---------------------------------------------------------------------
+
+(1 row)
+
+step s2-drop-type:
+ DROP TYPE my_type;
+ <waiting ...>
+step s1-commit: 
+    COMMIT;
+
+step s2-drop-type: <... completed>
+step s2-commit:
+ COMMIT;
+
+step s3-compare-snapshot:
+ SELECT count(*) = 0 AS same_metadata_in_workers
+ FROM
+ (
+  (
+   SELECT unnest(activate_node_snapshot())
+    EXCEPT
+   SELECT unnest(result::text[]) AS unnested_result
+   FROM run_command_on_workers($$SELECT activate_node_snapshot()$$)
+  )
+ UNION
+  (
+   SELECT unnest(result::text[]) AS unnested_result
+   FROM run_command_on_workers($$SELECT activate_node_snapshot()$$)
+    EXCEPT
+   SELECT unnest(activate_node_snapshot())
+  )
+ ) AS foo;
+
+same_metadata_in_workers
+---------------------------------------------------------------------
+t
+(1 row)
+
+
+starting permutation: s2-create-dist-func s1-begin s2-begin s1-start-metadata-sync s2-drop-dist-func s1-commit s2-commit s3-compare-snapshot
+step s2-create-dist-func:
+ CREATE FUNCTION squares(int) RETURNS SETOF RECORD
+    AS $$ SELECT i, i * i FROM generate_series(1, $1) i $$
+    LANGUAGE SQL;
+ SELECT create_distributed_function('squares(int)');
```

Comment:
minor: we dont need `create_distributed_function` anymore, all functions are distributed by default

Reply:
For some reason, I still need this call. Even after a rebase, this function is not distributed by default.

Spec file change: